commit 254e8a1efb2826ddcd18528dc01a76cf61b05ace
Author: hp <hp@138bc75d-0d04-0410-961f-82ee72b054a4>
Date:   Thu Apr 26 01:09:27 2018 +0000

    This appears to be present in compiler-rt upstream, but as part
    of more intrusive changes.  For gcc, the lack of this results in
    a fatal warning (-Werror) at build-time.
    
            * sanitizer_common/sanitizer_atomic_clang_other.h [_MIPS_SIM
            && _MIPS_SIM == _ABIO32] (lock): Add initializer for .pad member.
    
    
    git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@259663 138bc75d-0d04-0410-961f-82ee72b054a4

---
 libsanitizer/sanitizer_common/sanitizer_atomic_clang_other.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libsanitizer/sanitizer_common/sanitizer_atomic_clang_other.h
+++ b/libsanitizer/sanitizer_common/sanitizer_atomic_clang_other.h
@@ -35,7 +35,7 @@
 static struct {
   int lock;
   char pad[32 - sizeof(int)];
-} __attribute__((aligned(32))) lock = {0};
+} __attribute__((aligned(32))) lock = {0, {0}};
 
 template <class T>
 T __mips_sync_fetch_and_add(volatile T *ptr, T val) {
